### PR TITLE
Add stories for the subject viewer banners

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

Describe your changes:
This adds basic stories for the subject viewer banners before working on #932

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import styled from 'styled-components'
+import { AlreadySeenBannerContainer } from './AlreadySeenBannerContainer'
+import readme from '../../README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js]: If these are common to all stories, could they be moved to the storybook config? There is an ability to add global decorators: https://storybook.js.org/docs/addons/introduction/#storybook-decorators

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js]: Which background is the default varies for the story, so I don't think the global decorator will work in this case. Happy to be wrong though.

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const darkThemeConfig = Object.assign({}, config, { backgrounds: backgrounds.darkDefault })
+
+const subject = SubjectFactory.build({ already_seen: true })
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+
+storiesOf('Banners | AlreadySeenBannerContainer', module)
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/AlreadySeenBanner/AlreadySeenBannerContainer.stories.js]: The container / component abstraction doesn't make too much difference outside of the component; I'd simply call it `AlreadySeenBanner`

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/RetiredBanner/RetiredBannerContainer.stories.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+const darkThemeConfig = Object.assign({}, config, { backgrounds: backgrounds.darkDefault })
+
+const subject = SubjectFactory.build({ retired: true })
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+
+storiesOf('Banners | RetiredBannerContainer', module)
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/RetiredBanner/RetiredBannerContainer.stories.js]: Same here

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/UserHasFinishedWorkflowBanner/UserHasFinishedWorkflowBannerContainer.stories.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+const darkThemeConfig = Object.assign({}, config, { backgrounds: backgrounds.darkDefault })
+
+const subject = SubjectFactory.build({ user_has_finished_workflow: true })
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+
+storiesOf('Banners | UserHasFinishedWorkflowBannerContainer', module)
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/UserHasFinishedWorkflowBanner/UserHasFinishedWorkflowBannerContainer.stories.js]: Same here

File: packages/lib-classifier/src/components/Classifier/components/Banners/components/WorkflowIsFinishedBanner/WorkflowIsFinishedBannerContainer.stories.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+const darkThemeConfig = Object.assign({}, config, { backgrounds: backgrounds.darkDefault })
+
+const subject = SubjectFactory.build({ finished_workflow: true })
+
+const StyledBox = styled(Box)`
+  position: relative;
+`
+
+storiesOf('Banners | WorkflowIsFinishedBannerContainer', module)
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/Banners/components/WorkflowIsFinishedBanner/WorkflowIsFinishedBannerContainer.stories.js]: Same here

**Comment by srallen** [comment]
@rogerhutchings this is ready for review again.

**Pull request merged**